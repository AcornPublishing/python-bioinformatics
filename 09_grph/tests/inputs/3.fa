>Rosalind_1071
AGATGCCGGACGCGCGCACATATATAGACGCAGTTCGACGTCTCACACGATATTCACTCG
AGCAATAAGTCGCGTCACACTAACCTATGGCGC
>Rosalind_6708
TTCGGGGAAATCAACACCACCTTTTCCTTTGACCACAGGTCGGTGAGAATGCTTAACCAC
TTCCAACAGGCAGGTAGCGAG
>Rosalind_6245
TGCCGTTTACCCAGCGTCGAATTTAAAGGGGTGAGATCGTATGTATGACCGGGGCATTGC
TCTACAGCAGTTGCATTAAGCTG
>Rosalind_3213
TCAATTCAGAAGCACTCGTCCGTCCTGCGAGTTCAACGGGCCTGACGAGGCGAGGCGTCA
ATTTACACCGCTAACCCAGGGCAGGTTGAGGAAGGA
>Rosalind_9143
GGGGGGAATCCGGTCATGAGTCGGATCTCAGCCTTTTGGAATAGTTGAGGCGAATAGCTT
TGAGGATGCACAACCTGAGCT
>Rosalind_6156
GCAAGATCACACTGGCTCTGAGCACATCCGTCGATTAACCAGAAACTCGCAGAATTGCTC
GTGTGCGCACCCGCCCAGGCCAAAGACGA
>Rosalind_4963
TAGGACGATCCGCCTAGATTAGTCGATCATGGATCTATATGAATCTGATGGGGATTAACA
TTAGGTTCATGGATTTCACTCGTAGCTAAGAAC
>Rosalind_9110
CTCGATCGTGCTCGTTAGAAGCTTTTTCACGTGGGTTTGGGTAGCTTGCACGGGTTATAT
CAGATAAGTTAAGCGTTCACATGCG
>Rosalind_3679
CCGGTATAGTTGCGTGCCTCAAACGAGAAGGGAACCTGCAGGACCAATTGTCTCTGTTGG
ACGTGGAGCTCGTCCAGATTTTGATGATGATAGACGTGGT
>Rosalind_6969
GAGCATACCTCCTCAGCGAACTGGAGGTTGCCATTCCTATGGCCGGAAACCGCCTATAAG
CTCCGGTTTTTAGCCCGTAGCCTATTTTCACGATC
>Rosalind_7497
GCCGCCCCCTATTGAACACTTATAGATTCACCGCAATTGATAATCTCGTTTTCTATGGGT
CTCAACGCTAGACGACGAGATAGCATCTCTGATTCCTTTG
>Rosalind_4080
CCGAGGAGAACTTGTTATTGCGTCAAGTTTCCTTCGGCTTGCTGTCATGAGACCCTGGCT
TGTCCGAGACAATTCTAGAGTCCGCTCCCTTC
>Rosalind_9138
TGGTTTGTTCGCCTCTTCCACACTCCCGCGGGCGGATCCGTGGAGACCCACAGTAGGTTC
CCGTGTTCCTAATGCTGCGCACCG
>Rosalind_5365
TGACTTGTTGATTTCTTAACACCAATACAGACGCAGCGCGAAAAGAGTCGTGCAAGGCTG
AACTAATAACTATGCCCTGTACCGAAGCTATTGTCGT
>Rosalind_1078
ATCATGAGGCTTCTCGCCGCGTAAGGATGCCGTGCTAGGACGAACTATATGTCTCGACCG
AAAGAAAACCTACGAGATGC
>Rosalind_2014
TCGCGGCTCTTCTTAACTTTCCCGTGCCGGCGAGCGCCCGTAAAGCCCGGCAGTTGTCGA
GTACACATTAGACCTTCAATTA
>Rosalind_6983
ATCCACTGGACCCCTACCTTGTAAGAGTGAGGATGCCCTTCCCAAGTGAAAGGTGTAAGC
CCATAGGAGTATCCTTTATACCGC
>Rosalind_5581
CTAAGGCGCGTACTACTTAGCTGCGTAAATACGTGAGCGGCTGTCACCACATTTCTCGCA
GGTCTCTGAGTCTATGTCCG
>Rosalind_3545
AATCCCGTCGATTTGTACGCGATCCTACTCCCCTAACGTCCGCCCAATACAATGATAGGG
ATGGCCATCCTTATAATCCTGC
>Rosalind_1083
TCTTACGCAAGAAGCACCTATTACTCGATAGGAGAACTTATATGCGATCACCTTCACTTG
CCTAGCCGGCACAACTAAATACCTCTTCGCGTAT
>Rosalind_1032
AACCGGATAACCCACTTGCCAGTCACATGTCTCTGGTTCTTACCGTGCCGGCTTTTGCCC
CGTGAGACATTTACGCTGCGACTTGTGTATCGCTTGA
>Rosalind_3333
TTAAAAATTTTGGTTGTGGTGGGTTACAATAGGTACAATCCTAGATTAACATGTGGTTAA
GGTGCCAAAAATCATCCGAGGGAG
>Rosalind_7026
AAACAAAACGCACGCTTACACCTAAAGGGGGGTATCAGGTCGTGACGTGAGCATGCGCTT
CGGGCGATGCACGTTTCGAGAGCGGCGAAGCCCA
>Rosalind_2343
TAGCCAGGTGCGACGACCGGCAAAAACTGTCTTCACCTATGGGCAGAGGGGTGTCGAGTA
AACTAGGCAGGGGGTCCTGCATGTTGGTGAGGATGAG
>Rosalind_0610
GAGTAGTTGTCCACCTCAATCTCATTACGTCTGCCCGCCCTCATTTCGTCTCGTTATTAG
AATGACATGTACGGCCATGCTTCG
>Rosalind_6425
TTGACAATACTCATTTCTCGATTTACTCGCAGACCGTAAGTATACTAATTTTGTAGGCAC
TTCCACGGGGGGATGAACCTAAGACCCGC
>Rosalind_5221
GTGAGATCGACATAACCATTTGTACCGCTAAGACACCCTCGATCCGTGTCTTCGCGGGAC
TAATTAACACTTACGCAGGCCCGAGGGTAGA
>Rosalind_9524
TGGTGATTGATCACACCCGTGGAATACACTGTAATGGGAACGGTCGTCCGAGATGCTATC
TAACCGGAGGAACAGGCTGTTGGGGT
>Rosalind_2771
TAAAATATAACGCTGACTCAAGTTAAACTTGATGCAGTGGTGCCCTTTGGGACCTGTATG
CTATCTGAGACCCCGCATTGTCTCTACAACTACAGTTCCT
>Rosalind_8232
ATCCAGCCACTTACACTGCTTTGCCACTTACGATCGGACCCCAGACGGAAAAAAAATGCT
GAATGCGAACTAGTGCTGGTGCTGAGACGCGAGAA
>Rosalind_9056
GTAAGCCAAACCCTCAGGCTCAGGTCCGCTACTACTTTACTGTAAGTCATGCTCAACTGG
AAGGCCATGCGGGAAGCCTCTCTCAGAAGA
>Rosalind_1715
ACTGTCATCAATAGATGTGGAGCATACAGCGCTACCACGACCGTTTGCTTGTGCATTCAC
GTAGGTTGATTTGTTCTGCCC
>Rosalind_9475
TTAACCTGCGACTTGTAACTGTACCGTGTAGCGGTTAGCTCGGTGAGAGCATACTAACTC
ACTTTGAACGTTCGTTCTTGTGGGC
>Rosalind_4012
ACTACGCTTCACTCGAAGACTTCCCCTTACTCGCTAATTCTAGCCGGCGCGTTTTGATAT
CTTGAGCGCAGATCTGAACTGTAATTGGAGGATCACT
>Rosalind_8754
AGGATAATATCGGCATGGTAAGGGTGTCGGCCACATATTTAGCGATGCGTCGGTGAGAAA
ATACGCTCTGCTGAAGATAATGGGAATACAGCGCTG
>Rosalind_6165
CCTCCGTTAGTGATAACCTGCCAGGACGTTGGCGCTGCGAACTCTACTCAGCCCCTGTGT
CCTGCCATGAAATTCGCTATTCGGTGGCCTTGTACGTGC
>Rosalind_8270
GCGGCATAGTAAGCGCCTGCGGAAAGCCTGCTATACTGCCGCGAAACAGCTAAAGTATGA
GGTTCTGATCGATCGCCCCATACAGCTTATA
>Rosalind_8870
TCGTTGATCCAGGCTGGAAAGTTACTACAGATAGGAATACTGCCTATCATCATATGGATT
TAGCACCCGATTCTGGTTCGACACGAACTTCCCC
>Rosalind_4513
TGGTGACTCGTCTAACGCGGAGGTACCACATGGATCTGTCACCTGTATGCAATTGCCAGG
AATAGTACGATCGTGTAAAGTCCTGAG
>Rosalind_4509
CAAATGGGTGTGGCTTATGCTAGACCCTTTACGTACATACCCCTCATTTTTATATACTCG
CCACCTAGCTTTTCCTAGGTTTACCACAGTTCAC
>Rosalind_6938
GAAGAAGTAGGACGAATCTTAACCCTGTGACTCTTTCGAGGGCTCTCCCTAGTGTCTCGT
ATCTAAACCAACCCGATGTACTAGAGGTGGTTAGAGC
>Rosalind_5108
TTGAGGGACATGTGTGCGGAGTCACCCGATACGAATTTGTGCCTGATGCGATCCTAGATA
ACATGTAATTCCACCGACCTTAAAGGGTTAGGGCTT
>Rosalind_8452
CCGCATGTTCTATTGCTGCTGGTGTGTAATCAGCTAAACAAAACACTTATCCGGTAACTT
GCGGCCAACGAGCCTGAGTCCGAGATAGAACTAGACGAC
>Rosalind_4005
TTGGCACTTCTGTCACCTGTATGTGTATATGGCCCACCCGCGCGGGAAAATGGCCGCCAT
CCTTGATTTACTCTCGTACTCGGCTGATCGCTC
>Rosalind_0907
TCCCTGATGTGCAGATGTTCTGTTCTCAATTAAAGGAATAAACGAGCTCTAAAGAGACGC
TTGGCCTACTCATGAGTATG
>Rosalind_1181
GGATTGTTCTATGAGTGCCGCGCCTGCTCTGTTGAGTATCGCGTGATGATTCTTGTCTTG
TGATAGCCTTACCAGGCTCTAA
>Rosalind_2644
CTGATCGTCATTTAATATAGGTAATATGGGCTACTCCCCACCCCGACTCACGTCCCGTGG
GCCAACGTTTAAGGGATACGAT
>Rosalind_3393
CAGGAGCTTCTCATGTTGCCGAGAAACAGTACTAGCTGAGCCAAAATGGTAATTGGATCC
TTTGGCCACCTGATCGCCCCC
>Rosalind_7227
CTTGTATTTAGTGACCCGTTATCGTTGTTGGTCAGCCTGAGGGTAGCTCCACAATAATGT
AGCTCTGCCGTCCCAGAGCGATGGGGACTCTAGCA
>Rosalind_2772
TAACGAACTAATTGACAAAACCAGGAAGAACGGGAGGCTAGGCCATGCAAATGACGGCGA
CCCCATTTCTTAGTCCAAGTTGACGTTACTCT
>Rosalind_4771
CTCCTAATTGGGTTTATAGCGTTCTACGCCCCTCATATGATACGTGGAAAGGTCGTCTGC
GCGGGTTTGACGCCTCTTCTATCGATCA
>Rosalind_7444
CATAGGCCCATTGCGTAACGGCGCAAAGAACATCTACCACCAACCCTTGTTCCTAAGGGA
CTGATTGATCATCAACGCATGTAAACAGCTAAC
>Rosalind_4116
TCAGCCTCAGCATACTGAGCTAAGTTGTACCCTGCTCTAAGGTGCAAAAAGCTCCGTTAG
CAGTCGTGCCAGGTGGAACGCAG
>Rosalind_6895
CTCCCTCAATGGATTACTGGCCGCGGCTAGATCGCCACGCGTCTTTCCCGCGAAGCGAAT
CCGCACCATACCTCCGCATCACTGATGGTCGT
>Rosalind_4292
GGTGGGCGTCTCTCTCCCAACCCGATTGGGCTCCCGACGACCGGCGAAAAAGAGACCCGA
TGTGTAGCTTTTCGACCGTACGAGGCCTGTCAACACCTGC
>Rosalind_1998
CCAATATTACTAAGCATGACACCGACAGCGGAATGCAATGCGCCCCAGGCCTGTTGATAG
AATAATTCCGTAGTCTTGAATACTTGGAGGGGC
>Rosalind_0821
CTCTCAATCCCTAGGTTGAACATTTAACCCCGCTCGCTCATAAAAATGCATCATGTTCAG
ATGACAGCATCACTATTAAGTACCGTCCTGAGGG
>Rosalind_9503
CTGCAGGTGCGAGGCCATCGGAGTAGGCTGGATGGGTGCCCTGTTACCTCAGGTGGTAAG
TGATCGCAATTCCATGCCAGTCGATAGGTATAAA
>Rosalind_6125
AGCTGCTAATCGATTGCATTGACAACACGAAGCCTCACAAACAACTAGTCCTTCTTTAGA
GAATTACCCATAGTGTTGTTACCGC
>Rosalind_7895
GCGAGTCCAGACGTAACGGCCAGCGAAAATCGAGCATTTGGTTGGCGCCAACTACTACAC
ATATACTGTGTCATATCGACATCTACCAATATCCTG
>Rosalind_3662
TAACCCTATAAATACACTCAGAAACGAGCGGGTACAGCTCCAGTGTAATTAGGGAAAAGC
TTTTAACGCCTGAGTAGTACGAACACTTCGGTCGA
>Rosalind_3359
TTGGCCTAGTCTTACAAACCCTGTAGCGGCGACTTACAGGCTCAGTATTGTCCGAGCCAA
ATCTTGCCAACATGGAAGTTCAACGG
>Rosalind_8964
GATATGGAGCCACTTGCGCTGTACACCGATCTAACAAGCTTCTCTGTGCGGGTCGAAGAG
CCAATTCTCAAGGATGGGCCCGGTCCGGAATC
>Rosalind_2307
GACCCGAACGCCAGGCACGCACATCCTCCCTTTGGGTACGAGTAACAGGGTGCGTTGATC
TGTGATCACGTCTTCTCTCGCCGCACGGGGGTAATACGGC
>Rosalind_6131
ACAGCCATCGCGGTTCGAGCGCATATTTGTCCCGATATAAACTCACTAATAGAGTAGTTA
CAACGAGTTATCTCCCCCACCACAATA
>Rosalind_8442
ATGACTGCAAATCCCTTATTCCGCGAGAGACCCGAGGCCGCGGGGATGAGGTCACCTTTG
GCTTGGTTTACCAGCACTGCACGAGCGCCTGTTC
>Rosalind_3975
CAAAGAGAGTGCCTCTGATGGAGTGCGTGGCACCTACAATTTGTCTAGGGGGCATTTTAG
TCGCACAGCCCAGGAAGAGGCAC
>Rosalind_5025
GTTCTCTTACACCTTACTTCTGCCGTACAGCTATCCTGAATCAACGTCGGTCTAGTAGTG
AAATTCCATGGTATTAGTCTAGC
>Rosalind_0040
CATCGCGTTCGGGACGGCGATTTTTGTCGCGCCGTGTTGGCGGCTGAGCCCTAATTGCGC
CAGTGGACTTAATCTGCAGCCTTGTATTTCTACCCATGT
>Rosalind_0467
CGCGAACACGCTCCAGCACACTCCGCTGGACTCCACGAGCCATACTGCATACTAGCGCCG
CCCACAAGTACCGGCAATTTAGAAACTTCTT
>Rosalind_9909
CATTTCTACACAAGGACCCGGTATGTGCAAGGGTAGGGTGAGTGCTTTTTAATCTAGGTT
GCCTTTCACTCGGGCATTTTACCTTTCCAGTTAGTGGT
>Rosalind_9814
AGTCCTACTATCTCGGGAAAAATGTGTCAGCCTTGTGTTTGTATAATAATAGATCGAGAG
CATTGAGGACAGATACTGAAACTTTTTAAAAA
>Rosalind_9093
TCGCTTTCTACTGTGTAATCTCTCCAGCGATACAGTTCTATTCTGCAACACCAAGGCGGC
GTAACACATCAGGCTCAAGTCCCGCGGATGGTA
>Rosalind_2267
TGCACGGTTGGTGAATGCCCGACCGTAGTCGAGGGTCGTCAGTCTGAACTTAGATTTGTA
AAGCCGTACGTATGGGGCAAACCC
>Rosalind_2036
CTCCGGTCACGACCATTCCTCCGTTTGCAAATGGGCAAATGCAAGTGAGACTGGCCTTAA
TGGGGTAAACGCGACGGGGCCAACACGTCGAC
>Rosalind_4585
GTCCGAGAACCCCCGCCGTTTCGGGTCAATTTAGTTAGACGAGAGCGCGAACGGGCCTCC
GGGTAAATGTCATCGGACTTTG
>Rosalind_2350
CGTCTCGATAGAAGAAATGGTGTCGGGTCGCGTCTAACGTAAGGACACCGAGTTCCATGG
AAGATCCGCTCGATCGTCAACTATACTCATCG
>Rosalind_8935
TGAGTATGCGACCATTGTCCGCGCGGATCAGCGGTCGGCAGAGAGACCGTAGGTTATTGA
CAATGCACGGCATAGGGGGCTTACCACG
>Rosalind_4964
ATCGCCGCTCTTTAGAGACACCGCCTCTCGTCATCACCACCCAACGACTGGATCTCCACC
AAAAGATTTAAGAATGGTGTCTGAAAC
>Rosalind_9047
GTATAGCCTGGTGATCTACCTTCATGCGGGTGCTCACGGGGATTGATCGATGTTAACGAC
AAGCCAATTAATAAGGTTCACGAGAGAAT
>Rosalind_8162
GATCTCGTAGCGAATAGGATCTTGGGGTTCCTTGCGGCGTAGGAATCGGGGTGAGGCTAA
CCATTTACAATTACCTCAGCATGGGTGCCTCATGCGG
>Rosalind_4227
ACCCATGGCTCTTTCTGTCATAAGGATGTTAGCGGTCATCCACCATTAGCTGTGCTCTAG
CTAAAGCGTCTCTCACAATT
>Rosalind_7960
CAATATTCGGCTGGACCTTGTCGTAATAACCTCTCTCGGTCAGTAGTGCTTTTCACAGAG
CACAACCTAGGAGCGTCCAAGGAACG
>Rosalind_9039
GGGCTGGGCGGCTCATCTCGGTTTATGGCCGTTCCCGTATCGACCCCGTATGATGTCTAT
CAAGGATGTTCTAGTATGAAAGGGCACGGGG
>Rosalind_0070
TTCATTTGCTATATCCTTGAAATCCTTACGAGCTCTATCCTGGACGGCATCTGGAAGGCG
TGCCTCGAAAAAAGGCAATCTAGACTGT
>Rosalind_1532
ATCTGATAAACTACTTTTGAGACATGGCTGCATCAACGCTGGGGATTTGCCGCTACCGTC
AGTTCAGAGGTGTCTTGGGAGCGGCAGTCG
>Rosalind_4407
TATACCTCCCGTTATCTATCATCTGCTGCTCGGCGGCGGTAAGCTATACAGGCCGAAGCG
ACAGCAAAAGCGTCCGGTCGAATAGGGTATCATC
>Rosalind_9549
CATTTGAGAGAATTTTTACGAGAAGACCCTCCGACTCCTATATGGCTGACCGATAAGAAG
ATACTCACTCTCACTGCCACATCCTTCCCCCGCATACG
>Rosalind_8786
GCAGTACAGGTTAGTCGAGAGTGGGGTTAACAATAGGTCGCAGGGCTTTGCAATTATACG
AGACGAACTCCACGCCCTGATT
>Rosalind_7170
TACTTCACAATAGGCATTAAAGTGGGAATTTTGGCGAGGTACAATAGCGAAGGAAGTCTC
ATTGCAGACCCTAACCCTAATAGGCATT
>Rosalind_2102
GAAACATCGAGCGTGGGCGACCCTTCTTAAGTGGTATGTGTGATTGATAAACGTCTCCAA
GACAATTTCGGTTTAGATCAATCACTTGAGCA
>Rosalind_3612
GAGTAGGTTTTCAACGCTATAAACCATTAGCAGGCGGCGGATTAGGAAGTCTGCTTAGCT
GCGAAGTTTTATCGGTTGTCCTGTCATT
>Rosalind_1809
CATACGTCGAGCCTTCCTATTAGAGATGTCCACGGCGCGCGTGATTTAGAGATGTCTCGA
TTTCCTGCCCACTAGGTGTCGGCTA
>Rosalind_5201
GACTATTGTCGGTATAGTTCTTGGTCGCGAGTCCTCGGAATCGACAACAGTGTGAGTAAC
GACGAATTTACAATCTCGGCGATATGTCAACA
>Rosalind_1489
CCGGCTCGATATCATTCACAAGCAGGTTAGCCCGCCAAAGTCAGCAAATTTTCGGATCCC
TTTTGTAATCTGTGGGATTCGTC
>Rosalind_1865
CGTGGGTAAGCTCCGAACATCCAGACGCATTAATTCTCATTGATTATGCTGGAAAAGTGA
CACGCTCGACAGCTTTCAGCTCGTTCCT
>Rosalind_3260
TGACCACATAATAATTTGATTAGGAAAATTGCTGTACTCCATTAGGTTTGGAGAATGTGC
CCAAAAGACAATGGAAAGGTAATTATCT
>Rosalind_6724
GGTTAGCGTCATTGAAGCAGACACAATATAGGGACCGCATTATTCACACCGTATGGTCAA
GGTAACGTTTCGGATGTATCGA
>Rosalind_9595
AGTAATCATGCACCTGAGCTTGATACTCGCGGATATTGAAACTCCTGCCGCAGCACGGAA
CGTCCTATGTAACGGGCTTCCGGTGA
>Rosalind_5627
TCAACGATCTTGCTAAGCGGCCTAGGCCGGAATCGCGATGGTTGCCACACTACGGAGCAA
TGGTCGCTCCAGGCACCCCATTCTTAATAGG
