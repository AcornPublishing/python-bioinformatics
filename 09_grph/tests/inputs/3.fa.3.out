Rosalind_0467 Rosalind_7227
Rosalind_0610 Rosalind_2014
Rosalind_0610 Rosalind_8870
Rosalind_0610 Rosalind_9093
Rosalind_0821 Rosalind_9039
Rosalind_0821 Rosalind_9143
Rosalind_0907 Rosalind_8442
Rosalind_1032 Rosalind_3260
Rosalind_1032 Rosalind_5365
Rosalind_1032 Rosalind_8935
Rosalind_1071 Rosalind_0467
Rosalind_1078 Rosalind_2267
Rosalind_1078 Rosalind_6245
Rosalind_1083 Rosalind_4407
Rosalind_1181 Rosalind_2771
Rosalind_1181 Rosalind_2772
Rosalind_1181 Rosalind_3662
Rosalind_1489 Rosalind_4585
Rosalind_1532 Rosalind_2014
Rosalind_1532 Rosalind_8870
Rosalind_1532 Rosalind_9093
Rosalind_1809 Rosalind_5581
Rosalind_1865 Rosalind_6165
Rosalind_2014 Rosalind_3333
Rosalind_2014 Rosalind_9475
Rosalind_2036 Rosalind_2307
Rosalind_2036 Rosalind_5201
Rosalind_2102 Rosalind_6156
Rosalind_2102 Rosalind_8786
Rosalind_2343 Rosalind_0610
Rosalind_2343 Rosalind_3612
Rosalind_2343 Rosalind_6969
Rosalind_2350 Rosalind_2014
Rosalind_2350 Rosalind_8870
Rosalind_2350 Rosalind_9093
Rosalind_2644 Rosalind_8162
Rosalind_2644 Rosalind_8964
Rosalind_2771 Rosalind_6165
Rosalind_2772 Rosalind_1083
Rosalind_3213 Rosalind_1181
Rosalind_3260 Rosalind_1083
Rosalind_3333 Rosalind_0610
Rosalind_3333 Rosalind_3612
Rosalind_3333 Rosalind_6969
Rosalind_3545 Rosalind_2267
Rosalind_3545 Rosalind_6245
Rosalind_3679 Rosalind_4292
Rosalind_3679 Rosalind_6724
Rosalind_4005 Rosalind_0821
Rosalind_4005 Rosalind_2036
Rosalind_4005 Rosalind_4771
Rosalind_4005 Rosalind_6895
Rosalind_4005 Rosalind_9110
Rosalind_4012 Rosalind_1715
Rosalind_4080 Rosalind_0070
Rosalind_4080 Rosalind_6708
Rosalind_4116 Rosalind_3393
Rosalind_4292 Rosalind_2267
Rosalind_4292 Rosalind_6245
Rosalind_4407 Rosalind_1078
Rosalind_4407 Rosalind_1532
Rosalind_4407 Rosalind_4964
Rosalind_4407 Rosalind_6983
Rosalind_4407 Rosalind_8232
Rosalind_4513 Rosalind_0610
Rosalind_4513 Rosalind_3612
Rosalind_4513 Rosalind_6969
Rosalind_4585 Rosalind_3359
Rosalind_4585 Rosalind_4005
Rosalind_4585 Rosalind_5108
Rosalind_4585 Rosalind_6425
Rosalind_4771 Rosalind_3213
Rosalind_4771 Rosalind_4116
Rosalind_4771 Rosalind_5627
Rosalind_4963 Rosalind_1032
Rosalind_4964 Rosalind_1032
Rosalind_5025 Rosalind_6125
Rosalind_5108 Rosalind_7227
Rosalind_5201 Rosalind_6131
Rosalind_5221 Rosalind_1071
Rosalind_5365 Rosalind_1865
Rosalind_5365 Rosalind_2350
Rosalind_5581 Rosalind_1489
Rosalind_5581 Rosalind_3679
Rosalind_5581 Rosalind_4080
Rosalind_5581 Rosalind_8452
Rosalind_5627 Rosalind_8754
Rosalind_6125 Rosalind_0467
Rosalind_6165 Rosalind_2267
Rosalind_6165 Rosalind_6245
Rosalind_6245 Rosalind_2644
Rosalind_6245 Rosalind_9503
Rosalind_6425 Rosalind_0467
Rosalind_6708 Rosalind_0610
Rosalind_6708 Rosalind_3612
Rosalind_6708 Rosalind_6969
Rosalind_6895 Rosalind_1865
Rosalind_6895 Rosalind_2350
Rosalind_6938 Rosalind_6125
Rosalind_6969 Rosalind_1078
Rosalind_6969 Rosalind_1532
Rosalind_6969 Rosalind_4964
Rosalind_6969 Rosalind_6983
Rosalind_6969 Rosalind_8232
Rosalind_6983 Rosalind_0467
Rosalind_7026 Rosalind_1998
Rosalind_7227 Rosalind_6156
Rosalind_7227 Rosalind_8786
Rosalind_7444 Rosalind_1032
Rosalind_7497 Rosalind_3359
Rosalind_7497 Rosalind_4005
Rosalind_7497 Rosalind_5108
Rosalind_7497 Rosalind_6425
Rosalind_7895 Rosalind_2644
Rosalind_7895 Rosalind_9503
Rosalind_8232 Rosalind_2102
Rosalind_8232 Rosalind_6938
Rosalind_8442 Rosalind_0070
Rosalind_8442 Rosalind_6708
Rosalind_8452 Rosalind_2307
Rosalind_8452 Rosalind_5201
Rosalind_8754 Rosalind_2644
Rosalind_8754 Rosalind_9503
Rosalind_8964 Rosalind_1078
Rosalind_8964 Rosalind_1532
Rosalind_8964 Rosalind_4964
Rosalind_8964 Rosalind_6983
Rosalind_8964 Rosalind_8232
Rosalind_9039 Rosalind_9143
Rosalind_9047 Rosalind_3545
Rosalind_9056 Rosalind_1071
Rosalind_9093 Rosalind_9047
Rosalind_9093 Rosalind_9056
Rosalind_9110 Rosalind_7895
Rosalind_9110 Rosalind_8270
Rosalind_9138 Rosalind_1489
Rosalind_9138 Rosalind_3679
Rosalind_9138 Rosalind_4080
Rosalind_9138 Rosalind_8452
Rosalind_9503 Rosalind_7026
Rosalind_9524 Rosalind_4292
Rosalind_9524 Rosalind_6724
Rosalind_9595 Rosalind_3260
Rosalind_9595 Rosalind_5365
Rosalind_9595 Rosalind_8935
Rosalind_9814 Rosalind_7026
Rosalind_9909 Rosalind_4292
Rosalind_9909 Rosalind_6724
